### PR TITLE
LPS-89006 SF

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

File: modules/apps/layout/layout-type-controller/layout-type-controller-content/src/main/java/com/liferay/layout/type/controller/content/internal/display/context/FragmentsEditorDisplayContext.java [hunk 1]
```diff
@@ -721,6 +736,41 @@ private ItemSelectorCriterion _getURLItemSelectorCriterion() {
 		return _getWidgetCategoriesContexts(portletCategory);
 	}
 
+	private boolean _isUsed(Portlet portlet) {
```
Comment (ealonso) [inline modules/apps/layout/layout-type-controller/layout-type-controller-content/src/main/java/com/liferay/layout/type/controller/content/internal/display/context/FragmentsEditorDisplayContext.java:739]: @pavel-savinov Porque no buscamos en portlet preferences? para saber si lo usamos o no?

File: modules/apps/layout/layout-type-controller/layout-type-controller-content/src/main/java/com/liferay/layout/type/controller/content/internal/portlet/action/AddPortletMVCActionCommand.java [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	private LayoutPageTemplateStructureLocalService
+		_layoutPageTemplateStructureLocalService;
+
+	@Reference
+	private Portal _portal;
+
+	@Reference
+	private PortletLocalService _portletLocalService;
+
+	private class AddFragmentEntryLinkCallable
```
Comment (ealonso) [inline modules/apps/layout/layout-type-controller/layout-type-controller-content/src/main/java/com/liferay/layout/type/controller/content/internal/portlet/action/AddPortletMVCActionCommand.java:192]: @pavel-savinov Porque necesitamos esto si sólo hacemos una transacción, la de add?

**Comment by pavel-savinov** [comment]
Just started reviewing :)

**Comment by pavel-savinov** [comment]
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2088. [See changes here.](https://github.com/pavel-savinov/liferay-portal/compare/b1ad0591f506~2...b1ad0591f506)

**Comment by pavel-savinov** [comment]
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2089. [See changes here.](https://github.com/pavel-savinov/liferay-portal/compare/1ac41730cda8~2...1ac41730cda8)

**Comment by pavel-savinov** [comment]
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2091. [See changes here.](https://github.com/pavel-savinov/liferay-portal/compare/1061b6d58402~2...1061b6d58402)

**Pull request closed without merging**